src/domain/collaboration/callout/callout.resolver.mutations.ts

```diff
@@ -61,6 +75,63 @@ export class CalloutResolverMutations {
     return await this.calloutService.deleteCallout(deleteData.ID);
   }
 
+  @UseGuards(GraphqlGuard)
+  @Mutation(() => CommunicationMessageResult, {
+    description: 'Send a message on a Comments Callout',
+  })
+  @Profiling.api
+  async sendMessageOnCallout(
```

Comment:
Interesting that you have a seprate method for this, as in theory it should be possible to achieve the same effect with sending a message on a Comments directly. To discuss, but I think on reflection this makes sense. There is a strong argument then that the mutation to send a message directly on a Comments entity should be removed as we will need a similar mutation on the Aspect entity for sending a message.